index.html

```diff
@@ -9,21 +9,64 @@
   </head>
   <body>
     <header>
-      <h1>Awesome books</h1>
-      <div id="books-container"></div>
+      <nav class="navbar">
+        <div class="nav-branding">Awesome Books</div>
+
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="nav-link" id="list">List</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link" id="add-new">Add New</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link" id="contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="date">August 4th 2022, 20:00pm</div>
     </header>
     <main>
-      <form class="books-form" action="#">
-        <input id="title" type="text" placeholder="Title" />
-        <br />
-        <br />
-        <input id="author" type="text" placeholder="Author" />
-        <br />
-        <br />
-        <button class="add-btn" type="submit">Add</button>
-      </form>
-    </main>
+      <section class="books-list" id="all-book">
+        <h1>All Awesome books</h1>
+        <div id="books-container"></div>
```

Comment:
- [ ] Great job in styling your individual book lists. There is something small left. Kindly add  the alternating styling for the list items to have one as with a background color as white and the other one with a background color of lets say a darker shade of maybe gray.

You could consider having something like this:

```
#books-container .book-item:nth-child(odd) {
  background-color: rgb(223, 219, 219);
  color: #000;
}
```

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/183023183-287ff055-a23d-42ea-9232-84829c3c7640.png)

**This is the desired wireframe design:**

![image](https://user-images.githubusercontent.com/86318284/183023267-d78c7630-aac7-4478-8d98-ab4d614543ba.png)

Reply:
@Nemwel-Boniface Thank you again. I will fix this